packages/components/mention/src/mention-dropdown.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+  <div ref="dropdownRef" :class="ns.b('dropdown')">
+    <div v-if="$slots.header" :class="ns.be('dropdown', 'header')">
+      <slot name="header" />
+    </div>
+    <el-scrollbar
+      v-show="options.length > 0 && !loading"
+      ref="scrollbarRef"
+      tag="ul"
+      :wrap-class="ns.be('dropdown', 'wrap')"
+      :view-class="ns.be('dropdown', 'list')"
+    >
+      <li
+        v-for="(item, index) in options"
+        ref="optionRefs"
+        :key="item.value"
+        :class="[
+          ns.be('dropdown', 'item'),
+          ns.is('hovering', hoveringIndex === index),
+          ns.is('disabled', item.disabled || disabled),
+        ]"
+        @mouseenter="handleMouseEnter(index)"
+        @click.stop="handleSelect(item)"
+      >
+        <slot name="label" :item="item" :index="index">
+          <span>{{ item.label ?? item.value }}</span>
+        </slot>
+      </li>
+    </el-scrollbar>
+    <div v-if="loading" :class="ns.be('dropdown', 'loading')">
+      <slot name="loading"> {{ t('el.mention.loading') }} </slot>
+    </div>
+    <div v-if="$slots.footer" :class="ns.be('dropdown', 'footer')">
+      <slot name="footer" />
+    </div>
+  </div>
+</template>
+<script setup lang="ts">
+import { computed, nextTick, ref, watch } from 'vue'
+import { useLocale, useNamespace } from '@element-plus/hooks'
+import { scrollIntoView } from '@element-plus/utils'
+import ElScrollbar from '@element-plus/components/scrollbar'
+import { mentionDropdownEmits, mentionDropdownProps } from './mention-dropdown'
+
+import type { MentionOption } from './types'
+
+defineOptions({
+  name: 'ElMentionDropdown',
+})
+
+const props = defineProps(mentionDropdownProps)
+const emit = defineEmits(mentionDropdownEmits)
+
+const ns = useNamespace('mention')
+const { t } = useLocale()
+const hoveringIndex = ref(-1)
+
+const scrollbarRef = ref<InstanceType<typeof ElScrollbar>>()
+const optionRefs = ref<HTMLElement[]>()
+const dropdownRef = ref<HTMLElement>()
+
+const handleSelect = (item: MentionOption) => {
+  emit('select', item)
```

Comment:
If the option is disabled, selection should be prevented.
```suggestion
  if (item.disabled) return
  emit('select', item)
```

Reply:
done